### ISSUE TITLE
Create notices in FOLIO

### ISSUE THREAD

**Issue by CUSeif** [culibraries/folio#119]
Create notices in FOLIO that mirror notices currently generated in Sierra

- Do all Sierra notices have a similar notice in FOLIO?
- Do we need all notices that Sierra generates?
- Does FOLIO allow the same information to be placed in the notices?
- Does FOLIO allow for print notices and how does it interact with our current receipt printers

**Comment by libraryfemme** [comment]
Borrowing ILLs (for CU patrons from other libraries) have their own overdue notice for CU patrons in Sierra. This is because renewals cannot be completed in Sierra. They have to go through ILLiad instead, so special wording is required. I am checking with Lisa Tatum on the Lending ILL notices. @CUSeif

**Comment by CUSeif** [comment]
> This is because renewals cannot be completed in Sierra.

Do we know that this is true in FOLIO?

**Comment by libraryfemme** [comment]
@CUSeif Yes, at least for the ILLs that we borrow through OCLC. It *may* be possible in ReShare, but that is a ways away.

**Comment by jwfuller** [comment]
Circ Policy Pod has [documented the Sierra notices](https://docs.google.com/spreadsheets/d/1UkFzptISyjim915ME1EtiJijOWspKntFTin38kjxKmc/edit#gid=1265020370)

**Comment by katallm** [comment]
### Current Status of Notices

Updating this to note which notice policies are currently active, what they do, and which templates are completed (in a temporary version.)

#### Staff Slips
- [ ] Hold 
- [ ] Pick slip
- [ ] Request delivery
- [ ] Transit

#### Patron notice policies
- [ ] **Active**: All Loan and Request Notices
- [ ] Inactive: Send No Notices
